src/en/charms-storage.md

```diff
@@ -237,6 +238,53 @@ default values:
 juju deploy postgresql --storage pgdata=ebs,10G cache=ebs-ssd
 ```
 
+### Remove storage
+
+If you remove a unit that has storage, the storage is removed with the unit.
```

Comment:
I would put this as a note at the end, and let the following paragraph be the intro.
Also - is this actually what happens currently?